### PR TITLE
chore(backend): use asset id as liquidity account id

### LINKED ISSUE #215: Replace deterministic asset tigerbeetle ids
https://github.com/interledger/rafiki/blob/main/packages/backend/src/accounting/utils.ts#L3-L31

Use asset id as asset liquidity tigerbeetle account id (like how invoice, payment, and peer ids are used for tigerbeetle (liquidity) account ids).
Use asset unit as asset settlement tigerbeetle account id.

### PR DESCRIPTION
**Pull request by wilsonianb**
## Changes proposed in this pull request

- Use asset id as liquidity account id.
- Use asset unit as settlement account id.
- Do not look up Tigerbeetle account before deposit or withdrawal.
- Remove unused `accountingService.getAccount()`

## Context

- fixes #215 
- progress towards #225 (which will benefit from polymorphic account models)

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

**Review by sentientwaffle — COMMENTED** [review]
Can you add documentation in the code (similar to the description of https://github.com/interledger/rafiki/issues/215) describing which model ids map to which accounts in TB?

File: packages/backend/src/accounting/service.ts [hunk 1]
```diff
@@ -407,40 +318,16 @@ export async function createTransfer(
...
     {
       id,
-      sourceAccount: {
-        asset: {
-          unit,
-          account: AssetAccount.Settlement
-        }
-      },
-      destinationAccount,
+      sourceAccountId: account.asset.unit,
```
[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/backend/src/accounting/service.ts:329]: It doesn't clash with any other names.
I'm leaning towards waiting to renaming that whenever Rafiki needs to update for those breaking tigerbeetle(-node) changes

File: packages/backend/src/asset/model.ts [hunk 2]
```diff
@@ -10,4 +10,8 @@ export class Asset extends BaseModel {
 
   // TigerBeetle account 2 byte unit field representing account's asset
   public readonly unit!: number
+
+  public get asset(): Asset {
```
[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/backend/src/asset/model.ts:14]: Yeah, I was considering doing an [abstract base account class](https://github.com/wilsonianb/rafiki/pull/68/commits/c9869042275a352b6ff4217777518b13adedd8b2#diff-df5fde6529171c723a5e82b7f5b8eddfe08e441662b8c30255be383605aa6fdcR30-R34) that extends `BaseModel`, but I think `BaseModel`'s objection specific stuff would prevent that from working as a replacement for [`AccountOptions`](https://github.com/interledger/rafiki/pull/226/files#diff-72a5530c2c89f105f75e071479a382e8f8408459d37968fb93b546cca4783b31R28-R37).
Would I need a mixin to allow the other models to both extend `BaseModel` and implement an `AccountOptions`-esque interface?

[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/backend/src/asset/model.ts:14]: Nevermind, adding both `extend` and `implements` just works:
```
export class Asset extends BaseModel implements AccountOptions {
```

**Review by sentientwaffle — APPROVED** [review]

**Pull request merged**